Source/WebKit/Scripts/process-entitlements.sh

```diff
@@ -42,6 +42,7 @@ function mac_process_webcontent_entitlements()
 
 function mac_process_webcontent_captiveportal_entitlements()
```

Comment:
Since you're here, could you please rename `captiveportal` to `lockdown_mode` in this file?

Reply:
That's a good point, will fix!

Thanks for reviewing!